buildfiles/cmake/ConfigureCompiler.cmake

```diff
@@ -33,16 +33,14 @@ else()
 	add_compile_options(-fno-exceptions)
 	add_compile_options(-fstack-protector)
 
-	if (COMPILER_X86)
-		add_compile_options(-msse -msse2 -mcx16)
-	endif()
-
 	if (COMPILER_ARM)
 		if (APPLE)
 			add_compile_options(-march=armv8.4-a)
 		else()
 			add_compile_options(-march=armv8.1-a)
 		endif()
+	else()
```

Comment:
shouldnt this be an elseif (COMPILER_X86) ?

Reply:
Sure, why not. Maybe someday someone will port to risc-v.